src/pages/Search/SearchHoldReasonPage.tsx

```diff
@@ -18,11 +19,18 @@ function SearchHoldReasonPage({route}: PlatformStackScreenProps<Omit<SearchRepor
     const {translate} = useLocalize();
     const {backTo = '', reportID} = route.params ?? {};
     const context = useSearchContext();
+    const [reports] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}`, {canBeMissing: false});
```

Comment:
I see that we're passing `reportID` and `reports` as params, but all we do with those in `bulkHold` is `const iouReport = reports?.[`${reportID}`];`

Can we instead:

1. Change the Onyx connection to just get the specific report, i.e. `const [reports] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, {canBeMissing: false});`
2. Update the params of `bulkHold` to take the specific report, instead of all reports